Babylon/commands/api/solutions/service/api.py

```diff
@@ -0,0 +1,44 @@
+from typing import Optional
+
+from Babylon.utils.request import oauth_request
+
+
+class SolutionService:
+    def __init__(self, state: dict, azure_token: str, spec: Optional[dict] = None):
+        self.state = state
+        self.spec = spec
+        self.azure_token = azure_token
+        self.url = self.state["api"]["url"]
+        self.organization_id = self.state["api"]["organization_id"]
+        self.solution_id = self.state["api"]["solution_id"]
```

Comment:
would it this be a problem for get-all or create commands when we don't have a solution_id ? or it will be just ignored ?

Reply:
solution_id should just stay at None and be unused